db/schema.rb

```diff
@@ -37,16 +37,35 @@
     t.index ["user_id"], name: "index_poems_on_user_id", using: :btree
   end
 
+  create_table "submission_packets", force: :cascade do |t|
+    t.integer "poem_id",       null: false
+    t.integer "submission_id", null: false
+    t.string  "status"
+    t.index ["poem_id"], name: "index_submission_packets_on_poem_id", using: :btree
+    t.index ["submission_id"], name: "index_submission_packets_on_submission_id", using: :btree
+  end
+
+  create_table "submissions", force: :cascade do |t|
+    t.string  "title",        null: false
+    t.string  "status",       null: false
+    t.string  "submitted_to", null: false
+    t.integer "user_id",      null: false
+    t.index ["user_id"], name: "index_submissions_on_user_id", using: :btree
+  end
+
   create_table "users", force: :cascade do |t|
     t.datetime "created_at",                     null: false
     t.datetime "updated_at",                     null: false
     t.string   "email",                          null: false
     t.string   "encrypted_password", limit: 128, null: false
     t.string   "confirmation_token", limit: 128
     t.string   "remember_token",     limit: 128, null: false
-    t.index ["email"], name: "index_users_on_email", using: :btree
+    t.index ["email"], name: "index_users_on_email", unique: true, using: :btree
```

Comment:
It's okay to modify a migration that you're adding in a new branch. You just have to make sure to run/re-run it, so it produces the correct `db/schema.rb`. At this point, I would pull this out into a separate quick PR, explaining that this change was missed in a previous commit/PR.